pkg/agent/openflow/pipeline.go

```diff
@@ -1600,6 +1644,18 @@ func (f *featureNetworkPolicy) allowRulesMetricFlows(conjunctionID uint32, ingre
 			Done()
 	}
 	var flows []binding.Flow
+	// If the rule is ingress for IGMP or it is egress for multicast traffic, then this function generates flow to match
+	// conjunction ID and goto next table.
```

Comment:
I meant adding comments to explain the difference from normal traffic (metricFlow()) and why.

Reply:
done